code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -173,15 +173,15 @@ var/global/list/robot_modules = list(
 	src.emag = new /obj/item/weapon/melee/energy/sword(src)
 	return
 
-/obj/item/weapon/robot_module/medical

```

Comment:
Uh, no. This is not a general module. This would also break the parenting off of this. Leave this line as is, please.